pkg/controller/windowsmachineconfig/nodeconfig/nodeconfig.go

```diff
@@ -104,6 +109,21 @@ func (nc *nodeConfig) configureNetwork() error {
 	return nil
 }
 
+// applyWorkerLabel applies the worker label to the Windows Node we created.
+func (nc *nodeConfig) applyWorkerLabel() error {
+	if _, found := nc.node.Labels[WorkerLabel]; found {
+		log.Info("Worker label already present")
```

Comment:
Please reference the worker label as defined in the message. I am also not sure if it needs to be `Info`.